buildSrc/subprojects/build-update-utils/src/main/kotlin/gradlebuild/buildutils/tasks/SubprojectsInfo.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gradlebuild.buildutils.tasks
+
+import com.google.gson.GsonBuilder
+import gradlebuild.buildutils.model.GradleSubproject
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Internal
+import java.io.File
+
+
+open class SubprojectsInfo : DefaultTask() {
```

Comment:
```suggestion
abstract class SubprojectsInfo : DefaultTask() {
```

This should be `abstract`.

Even better would be to avoid task inheritance altogether. For me that is an antipattern which we want to reduce as much as we can. But I see why you did it here. Maybe it can stay like it is.